@Test(expected=RuntimeException.class) public void testExtractIdThrow() throws Exception {
}